integration-tests/pkgJson-restore.spec.js

```diff
@@ -31,6 +31,8 @@ var TIMEOUT = 240 * 1000;
 *   should initially be in pkg.json and/or config.xml.
 */
 
+/* eslint "promise/catch-or-return": "error" */
```

Comment:
Why put `eslint "promise/catch-or-return": "error"` here? Shouldn't it be in a .eslint config file (doesn't have to be top-level)?

Reply:
That's an oversight. Thanks, will remove!